### ISSUE TITLE
Crash looking up hunter's backpack recipe

### ISSUE THREAD

**Issue by bluelightning32** [slightlycolorblind/Wildcraft#40]
Repro:
1. Hit 'h' to go to the help.
2. Click on hunter's backpack
3. Observe the client side crash

```
Running on 64 bit Linux with 62 GB. 
Game Version: v1.18.0-rc.6 (Unstable)
4/15/2023 9:55:24 PM: Critical error occurred
Loaded Mods: game@1.18.0-rc.6, wildcraft@1.6.3, hudclock@2.9.0, prospectorinfo@4.1.5-pre, creative@1.18.0-rc.6, survival@1.18.0-rc.6
System.DivideByZeroException: Attempted to divide by zero.
  at Vintagestory.API.Client.SlideshowGridRecipeTextComponent.RenderInteractiveElements (System.Single deltaTime, System.Double renderX, System.Double renderY, System.Double renderZ) [0x00261] in <d687395392684752a1a04b19919ed72e>:0 
  at Vintagestory.API.Client.GuiElementRichtext.RenderInteractiveElements (System.Single deltaTime) [0x00087] in <d687395392684752a1a04b19919ed72e>:0 
  at Vintagestory.API.Client.GuiComposer.Render (System.Single deltaTime) [0x000b7] in <d687395392684752a1a04b19919ed72e>:0 
  at Vintagestory.API.Client.GuiDialog.OnRenderGUI (System.Single deltaTime) [0x0001c] in <d687395392684752a1a04b19919ed72e>:0 
  at Vintagestory.GameContent.GuiDialogHandbook.OnRenderGUI (System.Single deltaTime) [0x0006a] in <a7b261df1be64b299361a4beb49dc14f>:0 
  at Vintagestory.Client.NoObf.GuiManager.OnRenderFrameGUI (System.Single deltaTime) [0x00034] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.API.Client.DummyRenderer.OnRenderFrame (System.Single deltaTime, Vintagestory.API.Client.EnumRenderStage stage) [0x00000] in <d687395392684752a1a04b19919ed72e>:0 
  at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage (Vintagestory.API.Client.EnumRenderStage stage, System.Single dt) [0x00072] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage (Vintagestory.API.Client.EnumRenderStage stage, System.Single dt) [0x00028] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer (System.Single dt) [0x0005a] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.GuiScreenRunningGame.RenderToDefaultFramebuffer (System.Single dt) [0x0003e] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.ScreenManager.Render (System.Single dt) [0x001ec] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.ScreenManager.OnNewFrame (System.Single dt) [0x000cc] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame (System.Object sender, OpenTK.FrameEventArgs e) [0x00141] in <bbe608fcb6004efe90024778c1bf8392>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[OpenTK.FrameEventArgs].invoke_void_object_TEventArgs(object,OpenTK.FrameEventArgs)
  at OpenTK.GameWindow.OnRenderFrame (OpenTK.FrameEventArgs e) [0x00000] in <a4886e33f9fc416195d24950e7205b94>:0 
  at OpenTK.GameWindow.OnRenderFrameInternal (OpenTK.FrameEventArgs e) [0x00010] in <a4886e33f9fc416195d24950e7205b94>:0 
  at OpenTK.GameWindow.RaiseRenderFrame (System.Double elapsed, System.Double& timestamp) [0x0000c] in <a4886e33f9fc416195d24950e7205b94>:0 
  at OpenTK.GameWindow.DispatchRenderFrame () [0x00038] in <a4886e33f9fc416195d24950e7205b94>:0 
  at OpenTK.GameWindow.Run (System.Double updates_per_second, System.Double frames_per_second) [0x00104] in <a4886e33f9fc416195d24950e7205b94>:0 
  at OpenTK.GameWindow.Run () [0x00000] in <a4886e33f9fc416195d24950e7205b94>:0 
  at Vintagestory.Client.ClientProgram.Start (Vintagestory.Client.ClientProgramArgs args, System.String[] rawArgs) [0x00421] in <6fff5c2873464b009244947bd7cd751b>:0 
  at Vintagestory.Client.ClientProgram+<>c__DisplayClass16_0.<.ctor>b__0 () [0x00000] in <6fff5c2873464b009244947bd7cd751b>:0 
  at Vintagestory.ClientNative.CrashReporter.Start (System.Threading.ThreadStart start) [0x00007] in <bbe608fcb6004efe90024778c1bf8392>:0 
```

**Comment by subenji** [comment]
Came to report this too:
```
Running on 64 bit Windows with 32 GB RAM 
Game Version: v1.18.0-rc.6 (Unstable)
16/04/2023 21:08:09: Critical error occurred
Loaded Mods: game@1.18.0-rc.6, wildcraft@1.6.3, creative@1.18.0-rc.6, survival@1.18.0-rc.6
System.DivideByZeroException: Attempted to divide by zero.
   at Vintagestory.API.Client.SlideshowGridRecipeTextComponent.RenderInteractiveElements(Single deltaTime, Double renderX, Double renderY, Double renderZ) in VintagestoryApi\Client\UI\Elements\Impl\Interactive\Text\Richtext\SlideshowGridRecipeTextComponent.cs:line 314
   at Vintagestory.API.Client.GuiElementRichtext.RenderInteractiveElements(Single deltaTime) in VintagestoryApi\Client\UI\Elements\Impl\Interactive\Text\GuiElementRichtext.cs:line 492
   at Vintagestory.API.Client.GuiComposer.Render(Single deltaTime) in VintagestoryApi\Client\UI\GuiComposer.cs:line 711
   at Vintagestory.API.Client.GuiDialog.OnRenderGUI(Single deltaTime) in VintagestoryApi\Client\UI\Dialog\GuiDialog.cs:line 373
   at Vintagestory.Client.NoObf.GuiManager.OnRenderFrameGUI(Single deltaTime) in VintagestoryLib\Client\Systems\Gui\GuiManager.cs:line 302
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\Util\ClientEventManager.cs:line 201
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\ClientMain.cs:line 785
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt) in VintagestoryLib\Client\ClientMain.cs:line 963
   at Vintagestory.Client.GuiScreenRunningGame.RenderToDefaultFramebuffer(Single dt) in VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:line 241
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 722
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 625
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 119
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 476
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 452
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs)
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93

```

**Comment by gabriella-campos-davis** [comment]
Thanks for the report, will be fixed in 1.6.4
